frontend/src/pages/modelServing/screens/projects/NIMServiceModal/ManageNIMServingModal.tsx

```diff
@@ -122,14 +124,24 @@ const DeployNIMServiceModal: React.FC<DeployNIMServiceModalProps> = ({
   const [actionInProgress, setActionInProgress] = React.useState(false);
   const [error, setError] = React.useState<Error | undefined>();
   const [alertVisible, setAlertVisible] = React.useState(true);
-  const [pvcSize, setPvcSize] = React.useState<string>('30Gi');
+  const { pvcSize, setPvcSize, existingPvcSize, existingPVC } = usePVCSize(
+    projectContext?.currentProject.metadata.name,
+    editInfo?.inferenceServiceEditInfo,
+    editInfo?.servingRuntimeEditInfo?.servingRuntime,
+  );
+
+  const [createData, setCreateData, resetData] = useCreateStorageObjectForNotebook(existingPVC);
```

Comment:
`...forNotebook`... this is weird to use here.

What are you trying to make use of here? Why not use the data from usePVCSize?